pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// scorecardConfigKind is the default scorecard componentconfig kind.
+const ScorecardConfigurationKind = "ScorecardConfiguration"
```

Comment:
Since this is already in a `scorecard` group, we don't need `Scorecard` prefixes on the type names, IMO.

Reply:
I am inclined to agree.